### ISSUE TITLE
Ability to edit Regex hosts

### ISSUE THREAD

**Issue by xul8tr** [xcv58/Custom-JavaScript-for-Websites-2#73]
It would be nice to be able to edit regex hosts added to the list.

**Comment by xcv58** [comment]
The feature is dangerous because the user can easily mess up the existing setting and there is no way to revert the change. Let's say one has script A tie to regex_1, then the regex_1 be changed to regex_2. It's hard to restore it back to regex_1 if don't remember it exactly.

And there is a workaround for this, you can copy the script and open a new regex pattern and paste the codes.

**Comment by xul8tr** [comment]
Yes but then you get a bunch of random hosts in the lists. You have to test a bunch of regex's sometimes to get it to work on multiple environments with different url's and such. I think people who are using regex are advanced enough to not care about history. They will find the one that works again.
